### DISCUSSION TITLE
RAN metrics

### DISCUSSION THREAD

**Discussion by Mhdkb** [srsran/srsRAN_Project#293] [General Help]
Hello everyone,

I am using srsRAN version 23.10 and I would like to know if it is possible to get more RAN related metrics than the ones shown when running "t" on the console. 

Regards

**Comment by pgawlowicz** [comment]
The gnb can save metrics to the file if you add the following to its config file:
```
metrics: 
  enable_json_metrics: 1                                     # Optional BOOLEAN. Enables JSON metrics reporting. Supported: [0, 1]. 
  json_metrics_filename: gnb_metrics.json         # Optional TEXT. Sets the JSON metrics output path.     

```
